### ISSUE TITLE
Is it possible to get raw imu data from tracker?

### ISSUE THREAD

**Issue by pangfumin** [apollack11/vive_tracker_loc#1]
Hi , thank you for sharing this code.

What I have know is that you can get angles between vive tracker and lighthouse, if there a way to get imu data from tracker ? How about libsurvive?

**Comment by apollack11** [comment]
Yes libsurvive interfaces with the Vive components and is capable of getting IMU data. The IMU data is being stored in the Survive Object object. There is an example of pulling the IMU data in this file https://github.com/apollack11/vive_tracker_loc/blob/jarvis_edits/src/vive_pose_node.cpp

**Comment by jarvisschultz** [comment]
@apollack11 is correct: you can definitely get IMU data through libsurvive. As an additional note, I tried working with the IMU data a bit, and was not really sure what data I was actually seeing. I felt like it must have needed a calibration or something, but I never really looked into it. So just note that while you can access the IMU data, there might still be some work required to do anything useful with it.

**Comment by pangfumin** [comment]
@apollack11 @jarvisschultz  Thank you for your reply, I will try it.
